README.md

```diff
@@ -108,6 +108,8 @@ optional arguments:
   --entropy DO_ENTROPY  Enable entropy checks
   --since_commit SINCE_COMMIT
                         Only scan from a given commit hash
+  --rev_range FROM...TO       
```

Comment:
Should we consider a bit broader definition since the `rev` option rely on `git-rev-parse`, so we can pass `HEAD~` or `v1.5.1^0` or any other valid rev reference single or as a range.